Resources/Prototypes/_Shitmed/Entities/Debugging/debug_sweps.yml

```
@@ -0,0 +1,82 @@
+    damage:
+      types:
+        Slash: 200
+
+- type: entity
+  name: bang burner
+  parent: BaseItem
+  id: MeleeDebugBurner
+  description: burn yer parts
+  suffix: DEBUG
+  components:
+  - type: Tag
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)